### ISSUE TITLE
[BUG] querying a node API directly fails

### ISSUE THREAD

**Issue by surchs** [neurobagel/query-tool#138]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Expected Behavior

- `docker compose --profile local_node_query up -d` from the recipes
- go to the query tool (now pointing at a local n-API)
- attempt to query anything

expect it should succeed

### Current Behavior

The request arrives at the API and the API responds,
but the query tool crashes once it retrieves the response

### Error message

```
Unexpected Application Error!
Cannot read properties of undefined (reading 'sort')
TypeError: Cannot read properties of undefined (reading 'sort')
    at oj (http://localhost:3000/assets/index-ZYyfRoec.js:202:5051)
    at gh (http://localhost:3000/assets/index-ZYyfRoec.js:38:19519)
    at Xf (http://localhost:3000/assets/index-ZYyfRoec.js:40:3139)
    at ax (http://localhost:3000/assets/index-ZYyfRoec.js:40:44804)
    at rx (http://localhost:3000/assets/index-ZYyfRoec.js:40:39766)
    at mE (http://localhost:3000/assets/index-ZYyfRoec.js:40:39694)
    at $u (http://localhost:3000/assets/index-ZYyfRoec.js:40:39547)
    at ap (http://localhost:3000/assets/index-ZYyfRoec.js:40:35914)
    at ex (http://localhost:3000/assets/index-ZYyfRoec.js:40:34865)
    at k (http://localhost:3000/assets/index-ZYyfRoec.js:25:1565)
```

### Environment

- OS:
- Python/Node version:

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.2.2`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.2) :rocket:

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.2.2`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.2) :rocket:

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.2.2`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.2) :rocket:
